config/confd/templates/env.tmpl

```diff
@@ -18,7 +18,7 @@ VPN_BASE_MANAGEMENT_PORT=20000
 VPN_API_PORT=30000
 
 VPN_CONNECT_INSTANCE_COUNT={{ if eq (getv "/resin/vpn/production") "true" }}0{{ else }}1{{ end }}
-VPN_CONNECT_PROXY_PORT=3128
```

Comment:
This is going the change the listening port, is that your intention?

Reply:
I only realised after I put this in, when I started working on the `open-balena-api` PR, that I don't need to change this. Though having already done the work, it might make sense to merge it anyway, so that the flexibility is there in the future?

Reply:
OK, I've pushed an update where I split the listen and forward ports, since these could be different (in the case of k8s).